Makefile

```diff
@@ -1,13 +1,56 @@
-SHELL := /bin/bash
+SPACE := $(subst , ,)
+PACK = $(subst $(SPACE),,$(1))
+CMD_NOT_FOUND = $(error $(call PACK, $(1)) is required for this rule)
```

Comment:
Function to print an custom error when a command isn't not found.

Makefile can use function which are declared like normal variable, with the arguments being `$(1)`, `$(2)`, ...
Note that the `=` instead of `:=` is mandatory to not evaluate the function when declared